docs/migration_guide.rst

```diff
@@ -1,3 +1,37 @@
+0.X.X Migration guide
+=====================
+
+0.X.X makes the first step to remove the Cairo lang dependency!
+Some classes/functions from the Cairo lang are rewritten and are the part of our library:
+
+- transactions' dataclasses
+- `get_selector_from_name` and `get_storage_var_address` functions
+
+Deprecation
+-----------
+
+`compute_invoke_hash` is deprecated in favour of `compute_transaction_hash`.
+
+Breaking changes
+----------------
+
+1. `InvokeFunction` is replaced by the `Invoke` dataclass (behaviour is the same, just the name is changed).
+2. client_models.py does not contain:
+
+- Invoke,
+- InvokeFunction,
+- StarknetTransaction,
+- AccountTransaction,
+- ContractClass,
+- Declare,
+- DeployAccount.
+
+3. Transaction's `tx_type` field is renamed to `type`.
+4. The `types.py` is removed (outdated file containing only imports).
```

Comment:
Breaking changes should give a way of migration so that the old code works again.
That said, we probably should write from where one should import things now, instead of importing them from `types`.